### DISCUSSION TITLE
전체공간을 볼 수 있었으면 좋겠어요.

### DISCUSSION THREAD

**Discussion by joorijung** [metamong-friends/metamong-friends.github.io#30] [제안하기]
Quarter View에서 최대한 작게 보거나,  Shoulder View에서 각도 조절을 해봐도 전체공간을 다 볼 수는 없어서, 누가 어디에 있는지 한눈에 들어오지 않는 것이 조금 불편한 것 같아요. 전체를 조망할 수 있으면 좋을 것 같아요~!

+ 그리고 마우스 휠 클릭 했을 때 각도나 시야 방향 조절같은게 되면 좋겠어요,,ㅎ ㅎ

**Comment by merryblack** [comment]
안녕하세요! :) 의견 남겨주셔서 감사합니다!

혹시 화면(배경) 마우스 우클릭 후 드래그 해보셨을까요?
각도 및 시야 방향 조절이 우클릭으로 가능합니다 :)
이미 해당 방법으로 사용해보았으나 이 또한 제한적이라고 느껴지셨다면, 더 자유롭고 편리한 방법을 모색해 보겠습니다!
이에 관련하여 편리한 사용을 위해 조금 더 자세한 피드백 주시면 도움이 많이 될 것 같아요.
다시한번 의견 남겨주셔서 감사합니다!

**Reply by joorijung** [reply]
오 우클릭 기능이 있었네요 ㅎㅎㅎㅎ 감사합니다! 가이드를 다시 확인해볼게요!

**Reply by joorijung** [reply]
마우스 우클릭 하고 뗐을 때 원래대로 돌아오는데 그냥 내가 조절한대로 남아있으면 좋을것 같긴하네요..!

**Reply by ACT-TJ** [reply]
네 여러 동료분들이 말씀을 주시는 걸로 봐서 (쿼터뷰가 아닌 플랫한 정사각형 뷰, 돌린대로 남기) 유저들이 편안함을 
느끼는 각도가 좀더 다양한걸로 보여요. 저희 5~6월내 계획에 우클릭으로 돌린 각도를 유지하며 사용할 수 있는 기능
계획되어 있으니 릴리즈 정보 관심있게 봐주시면 좋을 것 같습니다. 의견 더 많이 주시구요^^ 관심 감사합니다.

**Comment by shlee303** [comment]
@joorijung 우클릭으로 회전하여 전체 맵을 볼 수 있도록 조치하였습니다. 추후 미니맵도 추가할 예정이에요~! 좋은 의견 감사합니다!
